src/formatDate.js

```diff
@@ -50,7 +50,26 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  const myIndex = 3;
```

Comment:
Ok, yet for constants we use uppercase.
And it's gonna be meaningful naming
`const MY_CONSTANT = "some-value";`